lambda_src/drivers/destination_s3.py

```diff
@@ -107,7 +110,7 @@ def write(
     )
 
     if not prefix:
-        prefixed_filename = f'{DATA_FOLDER_NAME}/{batch_id}/row-{row_index}.data.json'
```

Comment:
if we delete the references to `DATA_FOLDER_NAME`, we should delete the unused variable up top, as well. please set up pylint to tell you about things like this https://pylint.pycqa.org/en/latest/user_guide/messages/warning/unused-variable.html